webapp/src/Components/Replay/ReplayViewer/Viewer.tsx

```diff
@@ -13,9 +13,12 @@ import {
     Slider
 } from "replay-viewer"
 
+import { addFrameListener, FrameEvent } from "replay-viewer/eventbus/events/frame"
+import { doGet } from "../../../apiHandler/apiHandler"
 import { Replay } from "../../../Models"
 import { getReplayMetadata, getReplayViewerData, getReplayViewerDataRange } from "../../../Requests/Replay"
 import { LoadableWrapper } from "../../Shared/LoadableWrapper"
+import { AdvantageBarChart } from "./AdvantageBar"
 
 interface Props {
     replayId: Replay["id"]
```

Comment:
Can we add a prop for making advantage optional? i.e. if it's not a 1s replay don't download the predictions.